modules/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -0,0 +1,224 @@
+
+When you create an OpenShift cluster that uses the AWS Security Token Service (STS), you can customize your installation interactively. When you run `rosa create cluster --interactive` at cluster creation time, you are presented with a series of interactive prompts that enable you to customize your deployment.
+
+You can also select one of the following modes using the `--mode` option with the `rosa cluster create` subcommands:
+
+* `manual` mode. With this mode, you can generate the `aws` CLI commands needed to create the required AWS Identity and Access Management (IAM) roles and policies, and an OpenID Connect (OIDC) provider. The corresponding policy JSON files are also saved to the current directory. This enables you to review the details before running the `aws` commands manually. 
+* `auto` mode. You can use this mode to immediately create the required AWS Identity and Access Management (IAM) resources using the current AWS account. 
+
+[IMPORTANT]
+====
+Only public and AWS PrivateLink clusters are supported with STS. Regular private clusters (non-PrivateLink) are not available for use with STS.
+====
+
+.Prerequisites
+
+* You have completed the AWS prerequisites for ROSA with STS.
+* You have available AWS service quotas.
+* You have enabled the ROSA service in the AWS Console.
+* You have installed and configured the latest AWS, ROSA, and `oc` CLIs on your installation host.
+
+[NOTE]
+====
+link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-sharing.html[AWS Shared VPCs] are not currently supported for ROSA installations.
+====
+
+.Procedure
+
+. Create the required account-wide roles and policies, including the Operator policies:
++
+[IMPORTANT]
+====
+The account-wide roles and policies are specific to an OpenShift version, for example OpenShift 4.8. If you are using multiple cluster versions in one account, you can use `--prefix <prefix_name>` to specify a version-specific prefix for the Amazon Resource Names (ARNs). With the prefix, you can easily identify the roles and policies for each cluster version. If you do not specify a prefix, the `ManagedOpenShift` prefix is implied.
+====
++
+.. Generate the IAM policy JSON files in the current working directory and output the `aws` CLI commands for review:
++
+[source,terminal]
+----
+$ rosa create account-roles --mode manual --version 4.8 <1><2>
+----
+<1> `manual` mode generates the `aws` CLI commands and JSON files needed to create the account-wide roles and policies. After review, you must run the commands manually to create the resources.
+<2> You must specify an OpenShift release by using the `--version` option. The version that is installed when a ROSA cluster is created is determined by the roles and policies that are used.
++
+.. After review, run the `aws` commands manually to create the roles and policies. Alternatively, you can run the preceding command again using `--mode auto` to run the `aws` commands immediately.
```

Comment:
remove the word "again".  As it is it can be read to run this command twice one with manual then again with auto.  While it will work, no need to sew confusion.

Reply:
Thanks for highlighting this. Resolved.